test/viral_spiral/room/state_test.exs

```diff
@@ -58,14 +58,100 @@ defmodule ViralSpiral.Room.StateTest do
 
     test "world collapse", %{state: state} do
       state = state |> StateTransformation.update_room(%{chaos: 10})
-      assert {:over, :world} == State.game_over_status(state)
+
+      assert {:over, :world, %{bias_data: bias_data, affinity_data: affinity_data}} =
+               State.game_over_status(state)
+
+      assert %{most_biased_players: _, max_bias: _} = bias_data
+      assert %{most_affinity_players: _, max_affinity: _} = affinity_data
     end
 
     test "player win", %{state: state, players: players} do
       %{adhiraj: adhiraj} = StateTransformation.player_id_by_names(state)
```

Comment:
no need to call `player_id_by_names` again. 
This will suffice `%{adhiraj: adhiraj} = players`